src/main/java/org/spongepowered/common/mixin/core/item/MixinItemFishingRod.java

```diff
@@ -54,10 +54,12 @@ public ItemStack onItemRightClick(ItemStack itemStack, World world, EntityPlayer
             player.fishEntity.handleHookRetraction();
         } else {
             EntityFishHook fishHook = new EntityFishHook(world, player);
-            if (!Sponge.getGame().getEventManager()
+            if (world.isRemote || !Sponge.getGame().getEventManager()
```

Comment:
Could you make a ClientAware plugin marker interface for plugins to receive "client side client" events? Otherwise they would only receive the "server side" client events? Or would that be too messy?